src/main/java/mate/academy/springboot/datajpa/service/CategoryService.java

```diff
@@ -0,0 +1,14 @@
+package mate.academy.springboot.datajpa.service;
+
+import mate.academy.springboot.datajpa.model.Category;
+
+public interface CategoryService {
+    Category save(Category category);
+
+    Category get(Long id);
+
+    void delete(Long id);
+
+    Category update(Category category);
+
```

Comment:
remove the last empty line in all classes/interfaces/methods where it is present
```suggestion
```